podcasts/PodcastImage.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+import Kingfisher
+import PocketCastsServer
+
+struct PodcastImage: View {
+    let uuid: String
+    let size: Int
+
+    init(uuid: String, size: Int = 280) {
```

Comment:
Should we have this take a `PodcastThumbnailSize` instead of manually specifying the size?

```suggestion
    init(uuid: String, size: Int = 280) {
```

Reply:
Done in c3adc37cfc97454a4815fa92f1404c52678a2f5a